README.md

```diff
@@ -1,6 +1,27 @@
-The repository contains specification to manipulate General Transit
-Feed Specification (GTFS) and GTFS Realtime:
-* [GTFS](/gtfs/README.md)
-* [GTFS Realtime](/gtfs-realtime/README.md)
+# General Transit Feed Specification (GTFS) Overview
```

Comment:
The READMEs usually stick to one single top-level title # and keep the rest as ##, ###.